src/api/users/src/services/firestore.js

```diff
@@ -44,6 +62,8 @@ if (
   });
   logger.debug('Server running in online mode');
 } else {
+  // hardcode the environment variable if we're not using docker
+  if (!process.env.FIRESTORE_EMULATOR_HOST) process.env.FIRESTORE_EMULATOR_HOST = 'localhost:8088';
```

Comment:
No.  As discussed in the call on Friday, this needs to happen via the package.json, and you define it in your `npm dev` script.

Reply:
Sorry, forgot about this!